docs/command-line-flags.txt

```diff
@@ -0,0 +1,112 @@
+      --audit-log-version string                    API group and version used for serializing audit events written to log. (default "audit.k8s.io/v1")
+      --audit-policy-file string                    Path to the file that defines the audit policy configuration.
+      --audit-webhook-batch-buffer-size int         The size of the buffer to store events before batching and writing. Only used in batch mode. (default 10000)
+      --audit-webhook-batch-max-size int            The maximum size of a batch. Only used in batch mode. (default 400)
+      --audit-webhook-batch-max-wait duration       The amount of time to wait before force writing the batch that hadn't reached the max size. Only used in batch mode. (default 30s)
+      --audit-webhook-batch-throttle-burst int      Maximum number of requests sent at the same moment if ThrottleQPS was not utilized before. Only used in batch mode. (default 15)
+      --audit-webhook-batch-throttle-enable         Whether batching throttling is enabled. Only used in batch mode. (default true)
+      --audit-webhook-batch-throttle-qps float32    Maximum average number of batches per second. Only used in batch mode. (default 10)
+      --audit-webhook-config-file string            Path to a kubeconfig formatted file that defines the audit webhook configuration.
+      --audit-webhook-initial-backoff duration      The amount of time to wait before retrying the first failed request. (default 10s)
+      --audit-webhook-mode string                   Strategy for sending audit events. Blocking indicates sending events should block server responses. Batch causes the backend to buffer and write events asynchronously. Known modes are batch,blocking,blocking-strict. (default "batch")
+      --audit-webhook-truncate-enabled              Whether event and batch truncating is enabled.
+      --audit-webhook-truncate-max-batch-size int   Maximum size of the batch sent to the underlying backend. Actual serialized size can be several hundreds of bytes greater. If a batch exceeds this limit, it is split into several batches of smaller size. (default 10485760)
+      --audit-webhook-truncate-max-event-size int   Maximum size of the audit event sent to the underlying backend. If the size of an event is greater than this number, first request and response are removed, and if this doesn't reduce the size enough, event is discarded. (default 102400)
+      --audit-webhook-version string                API group and version used for serializing audit events written to webhook. (default "audit.k8s.io/v1")
+
+Features flags:
+
+      --contention-profiling   Enable lock contention profiling, if profiling is enabled
+      --profiling              Enable profiling via web interface host:port/debug/pprof/ (default true)
+
+Logging flags:
```

Comment:
For some flags of Logging, do we need to add labels that will be DEPRECATED?

```
Logging flags:

      --add_dir_header                                                                                                                               
                If true, adds the file directory to the header of the log messages (DEPRECATED: will be removed in a future release, see
                https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/2845-deprecate-klog-specific-flags-in-k8s-components)
      --alsologtostderr                                                                                                                              
                log to standard error as well as files (DEPRECATED: will be removed in a future release, see
                https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/2845-deprecate-klog-specific-flags-in-k8s-components)
      --log-flush-frequency duration                                                                                                                 
                Maximum number of seconds between log flushes (default 5s)
      --log_backtrace_at traceLocation                                                                                                               
                when logging hits line file:N, emit a stack trace (DEPRECATED: will be removed in a future release, see
                https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/2845-deprecate-klog-specific-flags-in-k8s-components)
                (default :0)
      --log_dir string                                                                                                                               
                If non-empty, write log files in this directory (DEPRECATED: will be removed in a future release, see
                https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/2845-deprecate-klog-specific-flags-in-k8s-components)
      --log_file string                                                                                                                              
                If non-empty, use this log file (DEPRECATED: will be removed in a future release, see
                https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/2845-deprecate-klog-specific-flags-in-k8s-components)
      --log_file_max_size uint                                                                                                                       
                Defines the maximum size a log file can grow to. Unit is megabytes. If the value is 0, the maximum file size is unlimited. (DEPRECATED:
                will be removed in a future release, see
                https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/2845-deprecate-klog-specific-flags-in-k8s-components)
                (default 1800)
      --logtostderr                                                                                                                                  
                log to standard error instead of files (DEPRECATED: will be removed in a future release, see
                https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/2845-deprecate-klog-specific-flags-in-k8s-components)
                (default true)
      --one_output                                                                                                                                   
                If true, only write logs to their native severity level (vs also writing to each lower severity level) (DEPRECATED: will be removed in a
                future release, see
                https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/2845-deprecate-klog-specific-flags-in-k8s-components)
      --skip_headers                                                                                                                                 
                If true, avoid header prefixes in the log messages (DEPRECATED: will be removed in a future release, see
                https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/2845-deprecate-klog-specific-flags-in-k8s-components)
      --skip_log_headers                                                                                                                             
                If true, avoid headers when opening log files (DEPRECATED: will be removed in a future release, see
                https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/2845-deprecate-klog-specific-flags-in-k8s-components)
      --stderrthreshold severity                                                                                                                     
                logs at or above this threshold go to stderr (DEPRECATED: will be removed in a future release, see
                https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/2845-deprecate-klog-specific-flags-in-k8s-components)
                (default 2)
  -v, --v Level                                                                                                                                      
                number for the log level verbosity
      --vmodule moduleSpec                                                                                                                           
                comma-separated list of pattern=N settings for file-filtered logging
```